package.json

```diff
@@ -119,34 +114,44 @@
 			"jsx",
 			"node"
 		],
-		"setupTestFrameworkScriptFile": "<rootDir>/scripts/testSetup.js"
+		"setupFilesAfterEnv": [
+			"<rootDir>/scripts/testSetup.js"
+		],
+		"snapshotSerializers": [
+			"enzyme-to-json/serializer"
+		]
 	},
 	"babel": {
 		"presets": [
 			[
-				"env",
+				"@babel/preset-env",
 				{
 					"modules": "commonjs"
```

Comment:
if you want it transpiled to umd just change this. We currently don't utilize the UMD building at all, I suggest we remove it.